### DISCUSSION TITLE
How to deal with "Host not enough" error

### DISCUSSION THREAD

**Discussion by Bitnine-sollee** [vesoft-inc/nebula#5860] [Q&A] [answered]
I am a newbie with NebulaGraph.
I am trying to create the Graph Space, and I am faced with the "Host not enough" error.
I have 2 questions.
1. Why is this error occurred?
2. What is the meaning of Vid Type : FIXED_STRING and INT64.

Thank you.

**Comment by QingZ11** [comment, marked as answer]
The "host not enough" error can be resolved using this FAQ. Link: https://docs.nebula-graph.io/3.6.0/20.appendix/0.FAQ/#how_to_resolve_the_error_host_not_enough

As for what vid type is, as its name shows, one is a fixed-length string type, and the other is an integer value type. However, this integer is int64, not int32 or int8.
